### PR TITLE
Branch fix ped bugs

### LINKED ISSUE #139: [PE-D] User Guide does not describes the trimming of whitespaces-lead parameters
The SUT trims the leading whitespaces in parameters, while the user guide does not mention this.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: zhXchD/ped#5

### LINKED ISSUE #148: [PE-D] Unable to create food with long descriptions
The user guide provided did not mention any minimum nor maximum length for descriptions of food.

Hence, the user should be able to input any length of description when creating a food.

However, when I tried to add a food with a long description, an error message occurs.

Command input:
add d/apple banana raspberry strawberry durian orange pie e/11-10-2020 q/2 p/medium t/frozen t/$15 t/contains nuts

Output:

The output shows that there is a maximum of 50 characters for food descriptions and does not accept my input. However, it should accept my input, since the user guide did not specify that descriptions are restricted to 50 characters. Perhaps this team should have specified such restrictions on the user guide.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: iamjamestan/ped#2

### LINKED ISSUE #151: [PE-D] UG not enough diagrams
Some sections of the UG did not have and screenshots or diagrams at all. This makes it inconvenient for first time users to follow through the user guide.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: iamjamestan/ped#5

### LINKED ISSUE #153: [PE-D] Certain sections of Table of Contents are too long
Features sections of Table of Contents appear too long. Perhaps you could try splitting the features further into subsections (adding, sorting, etc).

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: siangernlow/ped#1

### LINKED ISSUE #154: [PE-D] Missing Back to top button
Under the section for "Redoing previously undone command", there is no `^ Back to top` button. This section should also have it to maintain consistency throughout the User guide.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: siangernlow/ped#2

### LINKED ISSUE #155: [PE-D] Supplementary information needed for Quick Start
Under the Quick Start section, it mentions that an app window should be visible after opening the jar file. However, the app also additionally has another window describing expired items. It would be good if this section included this.

_Missing information on additional window_

_The window in question_

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: siangernlow/ped#3

### LINKED ISSUE #158: [PE-D] Function of ellipsis in the UG
Under the features sections, the format for the parameters are explained. However, it does not explain what the ellipsis in the formats mean, which may confuse the reader.

_The section that is mentioned_

_Ellipsis that need to be explained_

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: siangernlow/ped#6

### LINKED ISSUE #169: [PE-D] Documentation unclear on adding duplicate items
Related is the section in the UG. Note there is nothing that states adding duplicates is prohibited, only what a duplicate is.

When adding a duplicate item:

Expected behaviour:

Error message (or UG) should inform me that I cannot add duplicate items.

Another nice feature would be to add the qty to the existing item, but i guess that's not trivial (and may introduce more bugs)

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: JoeyChenSmart/ped#4

### PR DESCRIPTION
**Pull request by sagarsureka**
Add diagram to add command example.
Improve documentation on duplicate food items.
Add function of ellipsis.
Add information about expired items pop up in quick start.
Add missing back to top button in redo field.
Add documentation for description and tag length constraints.
Add documentation on trimming of whitespaces.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -132,6 +142,8 @@ You can then access the food item later on for editing, deleting etc.
 **Format:** `add d/DESCRIPTION e/EXPIRY_DATE q/QUANTITY [p/PRIORITY] [t/TAG]…`
 
 * A food item with the same description, expiry date and tags as another food item is considered a duplicate.
+* Description cannot contain more than 50 characters, including spaces.
+* Tags cannot contain more than 72 characters, including spaces.
```
Comment (jonahtanjz) [inline docs/UserGuide.md]: Perhaps can say "A tag cannot contain..." instead to be clearer because the restriction is for 1 tag?

**Review by jonahtanjz — COMMENTED** [review]
1 suggestion else LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (sagarsureka) [reply docs/UserGuide.md]: Thanks!

**Review by jonahtanjz — APPROVED** [review]
LGTM

**Pull request merged**